tests/RegressionTests_orion.intel.log

```diff
@@ -70,5516 +70,6 @@ Checking test 001 fv3_ccpp_control results ....
 Test 001 fv3_ccpp_control PASS
```

Comment:
Please rerun RT test, the log file only has test 001.  while we should have 108 test results.